tests/functional/centos/7/bs-lvm-osds/Vagrantfile

```diff
@@ -0,0 +1,542 @@
+# -*- mode: ruby -*-
```

Comment:
You should symlink this ``Vagrantfile``.

Reply:
this has been fixed